src/pages/Dashboard/MintRepayVai/MintVai/index.spec.tsx

```diff
@@ -123,7 +123,7 @@ describe('pages/Dashboard/MintRepayVai/MintVai', () => {
     expect(openSuccessfulTransactionModal).toHaveBeenCalledWith({
       transactionHash: fakeTransactionReceipt.transactionHash,
       amount: {
-        tokenId: 'xvs',
+        tokenId: 'vai',
```

Comment:
I don't have a preference but on my PR it was suggested we grab this from the Tokens constants